src/custom.json

```diff
@@ -0,0 +1,16 @@
+{
+    "title": {
```

Comment:
This file can have a few structures:

1. By Feature
```json
{
  "hide": ["col1", "col2"]
}
```

2. keys by column name - like what you did
3. Array of definitions, similarly to what ag-grid has

```json
{
   "column_definitions" : [
       {
          "name": "col1",
          "hidden": true,
          "side_filter": true
       }
   ]
}
This way you can define a typescript interface for this json object, and use it in your code to validate the input.

Reply:
Thanks, I think current structure looks simple and good. I will keep it for now.